enzyme/Enzyme/GradientUtils.cpp

```diff
@@ -2037,7 +2037,7 @@ bool GradientUtils::shouldRecompute(const Value *val,
 
 GradientUtils *GradientUtils::CreateFromClone(
     EnzymeLogic &Logic, Function *todiff, TargetLibraryInfo &TLI,
-    TypeAnalysis &TA, DIFFE_TYPE retType,
+    TypeAnalysis &TA, DerivativeMode mode, DIFFE_TYPE retType,
     const std::vector<DIFFE_TYPE> &constant_args, bool returnUsed,
     std::map<AugmentedStruct, int> &returnMapping) {
   assert(!todiff->empty());
```

Comment:
This shouldn't need to have a mode passed as it always must be ReverseModeAugmented

Reply:
done